store/precomputed_key/s3/s3.go

```diff
@@ -58,13 +57,13 @@ type Store struct {
 	stats            *store.Stats
 }
 
+func isGoogleEndpoint(endpoint string) bool {
+	return strings.Contains(endpoint, "storage.googleapis.com")
+}
+
```

Comment:
seems fine. if it's always going to be https://storage.googleapis.com, maybe trim the protocol instead of doing this?

Reply:
So right now it requires to be submitted without `https` for minio/client (i.e.: `--s3.endpoint=storage.googleapis.com`).

Another solution would be ensuring that the protocol is present before using `url.Parse()` and keep `s3utils.IsGoogleEndpoint`. In minio, they use a [helper method](https://github.com/minio/minio-go/blob/cca410398dd20409180d4ff2d017e34bea17c27c/utils.go#L107) for this.

Reply:
When/why do you ever need the full protocol url? And why does minio not allow it (I’ve noticed this also and found it annoying). I’m just curious at this point, but I’ll approve this, sounds fine for now

Reply:
So it's just needed for `url.Parse()` here. I think it may depend on each client implementation, but seems `minio-go` client don't handle if the endpoint has the protocol on it.